src/wallet/wallet2.cpp

```diff
@@ -3944,42 +3944,52 @@ void wallet2::process_unconfirmed_transfer(bool incremental, const crypto::hash
 // simplewallet does NOT update the pool info during automatic refresh to avoid disturbing interactive
```

Comment:
This comment is outdated because of how things work now with this PR, and should be modified to speak about how things were before this PR, or maybe deleted outright: Thanks to `m_secondary_pool_info_query_time` a call to this and using incremental update is not in danger to miss transactions anymore, like in my version that worked with a single time.